client/index.js

```diff
@@ -32,18 +33,16 @@ if(typeof __resourceQuery === "string" && __resourceQuery) {
 var hot = false;
 var initial = true;
 var currentHash = "";
-var logLevel = "info";
 var useWarningOverlay = false;
 var useErrorOverlay = false;
 
-function log(level, msg) {
-	if(logLevel === "info" && level === "info")
-		return console.log(msg);
-	if(["info", "warning"].indexOf(logLevel) >= 0 && level === "warning")
-		return console.warn(msg);
-	if(["info", "warning", "error"].indexOf(logLevel) >= 0 && level === "error")
-		return console.error(msg);
-}
+var INFO = "info";
+var WARNING = "warning";
+var ERROR = "error";
+var NONE = "none";
```

Comment:
The specific lines you highlighted are not duplicating the `quiet` functionality, but absolutely our consumption of a log level of `none` (see https://webpack.js.org/configuration/dev-server/#devserver-clientloglevel for the documentation specifying the use of `none`) most certainly seems like duplicate functionality. I think this is a good question for the Webpack team, being that we technically have two options which will do roughly the same thing - should one of these options be deprecated?

A proposed solution is to deprecate the use of `none` since that is technically not a log level (which the name `clientLogLevel` suggests), and leaving `quiet` for completely silencing output.